microsoft-edge/visual-studio-code/debugger-for-edge.md

```diff
@@ -1,17 +1,25 @@
 ---
-description: How to debug Microsoft Edge from Visual Studio Code
-title: Debug Microsoft Edge from Visual Studio Code
+description: You can debug Microsoft Edge from Visual Studio Code using the Debugger Extension
+title: Debug Microsoft Edge in Visual Studio Code
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 07/26/2021
+ms.date: 11/08/2021
 ms.topic: article
 ms.prod: microsoft-edge
 keywords: microsoft edge, web development, f12 tools, devtools, vs code, visual studio code, debugger
 ---
-# Debug Microsoft Edge from Visual Studio Code
+# Debug Microsoft Edge in Visual Studio Code
 
-[Visual Studio Code](https://code.visualstudio.com) includes a built-in debugger for Microsoft Edge which can launch the browser or attach to an already running browser.
+[Visual Studio Code](https://code.visualstudio.com) includes a built-in debugger for Microsoft Edge, which can launch the browser or attach to an already running browser.
 
-This built-in debugger lets you debug your front-end JavaScript code line-by-line and see `console.log()` statements directly from [Visual Studio Code](https://code.visualstudio.com).
+This built-in debugger lets you debug your front-end JavaScript code line-by-line and see `console.log()` statements directly from Visual Studio Code.
 
-For more information, navigate to [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging).
+For more information, see [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging) and [Browser debugging with Microsoft Edge DevTools integration in Visual Studio Code](/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension#browser-debugging-with-microsoft-edge-devtools-integration-in-visual-studio-code).
+
+
+<!-- ====================================================================== -->
+## Get in touch with the Debugger for Microsoft Edge Visual Studio Code Extension team
```

Comment:
Can we please get rid of this giant, distracting h2 (looming over this short page) and just have a low-key link instead

Reply:
i tried just removing the h2 altogether ... see what you think.